coordinator/src/routes/dispention/release.ts

```diff
@@ -0,0 +1,53 @@
+import * as fhir from "../../models/fhir/fhir-resources"
```

Comment:
Dispention?

Reply:
to mirror `/prescription`, not sure what a better path would be